rest/class-boldgrid-backup-rest-job.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * File: class-boldgrid-backup-rest-job.php
+ *
+ * @link       https://www.boldgrid.com
+ * @since      X.X.X
```

Comment:
you're using X.X.X in this file instead of SINCEVERSION ..... Just putting this here for ya so these don't get missed when you do your search and replace later

Reply:
See similar comment above.

Reply:
Thanks https://github.com/BoldGrid/boldgrid-backup/commit/fcaafd514a244a90807d28af823ba47f17d6233b